eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Epoch.hs

```diff
@@ -193,11 +173,13 @@ epochTransition = do
       TRC (epochState', UpecState pp (utxosPpups utxoSt'), ())
   let utxoSt'' = utxoSt' {utxosPpups = ppupSt'}
 
-  let Coin oblgCurr = obligation pp (rewards dstate') (psStakePoolParams pstate'')
-      Coin oblgNew = obligation pp' (rewards dstate') (psStakePoolParams pstate'')
+  let -- Coin oblgCurr = obligationDPState (DPState dstate' pstate'')
+      -- Now that obligation does not depend on the PParams at all, the current
+      -- and the new are identical, so we don't have to adjust for the difference.
+      Coin oblgNew = obligationDPState (DPState dstate' pstate'')
```

Comment:
is this left so that we can make use of it in a property test?

Reply:
I guess this was more of a comment explaining why certain things were deleted from the code. In the old system, changing the PParams could change what the obligation was.
Now it only depends on the DPState. So the code is much simpler. I have made this much cleaer in the comment.